server/src/test/scala/actors/PartitionedPortStateTestActor.scala

```diff
@@ -84,6 +84,7 @@ class PartitionedPortStateTestActor(probe: ActorRef,
     actor.ask(message).foreach { _ =>
       message match {
         case splits: SplitsForArrivals if splits.splits.keys.nonEmpty =>
+          println(s"Received splits for ${splits.splits.keys.size} flights")
```

Comment:
it is test , if you want to keep println then I guess it fine .

Reply:
oops i'd forgotten to push a commit to remove this :)